WcaOnRails/app/views/competitions/index.html.erb

```diff
@@ -8,7 +8,7 @@
     <%= render 'index_form' %>
   <% end %>
 
-  <div id="search-results" class="row competitions-<%= params[:display] == "list" ? "list" : "map" %>">
+  <div id="search-results" class="row competitions-<%= params[:display] %>">
```

Comment:
As before: !!!!!!INJECTION ATTACK!!!!!!

Maybe we should stop using `params[:display]` and instead create a *safe* variable called `@display`?